src/app/pages/system/support/support.component.ts

```diff
@@ -47,7 +47,20 @@ export class SupportComponent  {
       type: 'paragraph',
       name: 'support_text',
       paraText: this.sanitizer.bypassSecurityTrustHtml(
-        "Search https://redmine.ixsystems.com/projects/freenas to ensure the issue has not already been reported before filing a bug report or feature request. If an issue has already been created, add a comment to the existing issue. Please visit http://www.ixsystems.com/storage/ for enterprise-grade storage solutions and support. <br><br> A Redmine account is required to file an issue. Make an account at https://redmine.ixsystems.com/account/register. Use a valid email address when registering to receive issue status updates.")
+        'Search the <a \
+         href="https://redmine.ixsystems.com/projects/freenas" \
+         target="_blank" style="text-decoration:underline;"> \
```

Comment:
Don't break lines around an HTTP target.  That leaves whitespace in the link.
So:
style="text-decoration:underline;">FreeNAS issue tracker</a>\